data.sql

```diff
@@ -12,3 +12,26 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Boarmon', '2005-06-07', 7, TRUE, 20.40);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Blossom', '1998-10-13', 3, TRUE, 17.00);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Ditto', '2022-05-14', 4, TRUE, 22.00);
+
+INSERT INTO owners (full_name,age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name,age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name,age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name,age) VALUES  ('Melody Pond', 77);
+INSERT INTO owners (full_name,age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name,age) VALUES ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon')
+INSERT INTO species (name) VALUES ('Digimon')
+
+UPDATE animals SET species_id = 2 WHERE TRIM(name) LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE TRIM(name) NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Jennifer Orwell') WHERE TRIM(name) LIKE 'Gabumon';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Jennifer Orwell') WHERE TRIM(name) LIKE 'Pikachu';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Bob') WHERE TRIM(name) LIKE 'Devimon';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Bob') WHERE TRIM(name) LIKE 'Plantmon';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Melody Pond') WHERE TRIM(name) LIKE 'Charmander';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Melody Pond') WHERE TRIM(name) LIKE 'Squirtle';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Melody Pond') WHERE TRIM(name) LIKE 'Blossom';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Dean Winchester') WHERE TRIM(name) LIKE 'Angemon';
+UPDATE animals SET owner_id = (SELECT ID FROM owners WHERE TRIM(full_name) LIKE 'Dean Winchester') WHERE TRIM(name) LIKE 'Boarmon';
```

Comment:
- [ ] @devendra-alt Great work adding the owners to the animals, however, you forgot to add the first insertion where Sam Smith owns Agumon.
![image](https://github.com/devendra-alt/vet-clinic-database/assets/61408860/1a67eaed-e195-463a-a5f3-f97c0baadb27)